engine/api/warning/warning_project_variable_test.go

```diff
@@ -573,3 +661,12 @@ func TestUnusedProjectVariableWarning(t *testing.T) {
 	test.NoError(t, errAfterDelete)
 	assert.Equal(t, 0, len(warnsDelete))
 }
+
+func TestToto(t *testing.T) {
+	re, err := regexp.Compile("cds\\.proj\\.[a-zA-Z\\-_]+")
```

Comment:
should use raw string (`...`) with regexp.Compile to avoid having to escape twice